recipes/blimpy/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
The source distribution already contains this license file, so it is not necessary to contain license manually.

Reply:
Done